### PR TITLE
gdu@5.1.0: Add manifest

### PR DESCRIPTION
**Pull request by rashil2000**
Disk usage analyzer with console interface written in Go

Ref: https://github.com/dodorz/scoop/blob/master/gdu.json

### CODE REVIEW

**Comment by telenieko** [comment]
Note that a PR for gdu is already open on lukesampson/scoop-extras#6098

**Comment by rashil2000** [comment]
How exactly does one measure 'popularity'? If we go by GitHub stars, gdu is more popular than a similar tool called [dua](https://github.com/Byron/dua-cli) (which is already in main bucket), and is just a year younger. 

I don't think popularity/age is the problem. This repository is effectively defunct, just look at the amount of trivial open PRs.

**Comment by telenieko** [comment]
There is no clear guidelines as far as I know.

I tend to check stars, stack overflow, or Google search results.

In any case I personally default to Extras

**Comment by rashil2000** [comment]
Yeah, at least that one seems to be regularly maintained, for now.

**Comment by rashil2000** [comment]
Sorry I don't know how to put hash in the autoupdate field, could you point me towards an example?

File: bucket/gdu.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+            "hash": "6b3bb98fce576b364de177aa63a33dbab7d911f4e7f7aa0b964574b62df5ecef"
+        }
+    },
+    "installer": {
+        "script": [
+            "$exe = (Get-ChildItem \"$dir\" 'gdu_*.exe' | Select-Object -First 1).FullName",
+            "Move-Item \"$exe\" \"$dir\\gdu.exe\""
+        ]
+    },
+    "bin": "gdu.exe",
```
Comment (stevehipwell) [inline bucket/gdu.json:18]:
```suggestion
    "version": "5.7.0",
    "architecture": {
        "64bit": {
            "url": "https://github.com/dundee/gdu/releases/download/v5.7.0/gdu_windows_amd64.exe.zip",
            "hash": "e21f1798bc44c6d2e65fc3b9c7d43c46ebb66e7eeac97d113327acaf8893d973"
        }
    },
    "bin": [
        [
            "gdu_windows_amd64.exe",
            "gdu"
        ]
    ],
```

File: bucket/gdu.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/dundee/gdu/releases/download/v$version/gdu_windows_amd64.exe.zip"
+            }
+        }
```
Comment (stevehipwell) [inline bucket/gdu.json:25]:
```suggestion
        },
        "hash": {
            "url": "$baseurl/sha256sums.txt"
        }
```

**Review by stevehipwell — CHANGES_REQUESTED** [review]
Please see suggestions for the updated version, how to use a different shim name and how to use hashes.

**Comment by rashil2000** [comment]
@stevehipwell made the changes, let me know if they're fine

**Review by stevehipwell — APPROVED** [review]

**Comment by rashil2000** [comment]
@stevehipwell Just wanted to ask something. This repo was inactive for a long while (7-8 months), and it seems to have become active again. That said, I've had some CLI tools manifests with me for a while (all of them fit the Main bucket's criteria), I was holding myself from sending any PRs. Should I open a PR for each one of them (8-9 manifests), or a single PR would do?

**Comment by stevehipwell** [comment]
There have been a couple of new maintainers added (see https://github.com/ScoopInstaller/Main/issues/2403). For me I'd strongly prefer a single PR for each tool to keep the commits consistent and tracable.

**Comment by rashil2000** [comment]
Ah, thanks for the heads up!

Will do as you say.

**Pull request merged**